Sources/ProjectDescription/Dependencies/Pod.swift

```diff
@@ -0,0 +1,206 @@
+import Foundation
+
+/// A Pod dependency.
+///
+/// A Pod dependency can be either:
+///     - remote: A Git URL to the source of the package,
+///     and a requirement for the version of the package.
+///     - local: A relative path to the package.
+public enum Pod: Equatable, Codable {
+    case remote(url: String, requirement: Requirement)
```

Comment:
I think this one should be:

```swift
.local("Alamofire", path: "path/to/Alamofire")
.remote("Alamofire", requirement: .upToNextMajor("3.2.1"))
```

What do you think @danyf90 / @laxmorek ? Here's the [`Podfile`](https://guides.cocoapods.org/syntax/podfile.html) reference.

Reply:
IMO enough to start with